datadog_lambda/tracing.py

```diff
@@ -55,6 +55,11 @@
 
 dd_trace_context = {}
 dd_tracing_enabled = os.environ.get("DD_TRACE_ENABLED", "false").lower() == "true"
+if dd_tracing_enabled:
+    # Enable the telemetry client if the user has opted in
+    if os.environ.get("DD_INSTRUMENTATION_TELEMETRY_ENABLED", "false").lower() == "true":
```

Comment:
Lint check is failing
```suggestion
    if (
        os.environ.get("DD_INSTRUMENTATION_TELEMETRY_ENABLED", "false").lower()
        == "true"
    ):
```